Modules/xSharePoint/DSCResources/MSFT_xSPAddInManagementServiceApp/MSFT_xSPAddInManagementServiceApp.psm1

```diff
@@ -0,0 +1,104 @@
+function Get-TargetResource
```

Comment:
Is this actually named "Addin Management Service App" in central admin for a farm that is patched up to date? I thought it was still called App management there, so I don't want to be confusing in the language here if it still says that in central admin